.rubocop.yml

```diff
@@ -39,6 +39,9 @@ Layout/IndentationWidth:
   Exclude:
     - 'lib/puppet/provider/package/*.rb'
 
+Layout/LineEndStringConcatenationIndentation:
+  Enabled: true
+
```

Comment:
I think the default is to enable the cop, so maybe it doesn't need to be explicitly enabled here

Reply:
I believe that for new cops, they're set as pending until they're explicitly enabled: https://docs.rubocop.org/rubocop/versioning.html#pending-cops

When I run Rubocop locally, I get this:

```
$ bundle exec rubocop
The following cops were added to RuboCop, but are not configured. Please set Enabled to either `true` or `false` in your `.rubocop.yml` file.

Please also note that you can opt-in to new cops by default by adding this to your config:
  AllCops:
    NewCops: enable
```

Followed by a list of pending cops

Reply:
Ahh ok that makes sense